### PR TITLE
Allow page components

### PR DESCRIPTION
**Pull request by runnez**
https://www.notion.so/osome/Project-structure-3fd78dd663c34535bda1a152b31551f1

### CODE REVIEW

File: src/frontend.md [hunk 1]
```diff
@@ -15,18 +15,16 @@ src/
...
+      BankAccountList/
         [no nested directories]
         [avoid index.ts]
         [other files, but component files .tsx, styled.tsx, stories.tsx, local ui hooks, .test.ts, spec.ts, local static files should be placed in queries or services]
-        BankAccountTable.ts
-        BankAccountTable.styled.ts
-        useBankAccountTable.ts (ui hook used only in the current component)
-        BankAccountTable.stories.tsx
-        BankAccountTable.test.ts (unit tests)
-        BankAccountTable.spec.ts (integration tests)
```
Comment (Rendalf) [inline src/frontend.md:27]: `.spec.ts` is allowed too

File: src/frontend.md [hunk 2]
```diff
@@ -37,14 +35,16 @@ src/
     [exposed MF modules]
     [no nested directories]
   pages/
-    bankAccount/
-      BankAccountList/
+    bankAccount/ (entity, module, .etc)
+      BankAccountTable/
+        [no nested directories]
+        [components can be placed in the page/:folder if it is used only by components in this folder]
```
Comment (Rendalf) [inline src/frontend.md:41]: i guess we should avoid it, here is described why: https://github.com/OsomePteLtd/principles/pull/248#discussion_r1181506411

Comment (divineforest) [reply src/frontend.md:41]: agree

Comment (runnez) [reply src/frontend.md:41]: I believe it's a wrong guess I will show it why

File: src/frontend.md [hunk 3]
```diff
@@ -53,6 +53,8 @@ src/
         BankAccountList.page.ts
         BankAccountList.page.styled.tsx
         BankAccountList.page.spec.ts (integration tests)
+        [partials of the page can be placed nearby]
+        BankAccountListFilters.tsx
```
[the remark this answers is not in the document]
Comment (runnez) [reply src/frontend.md:57]: I would offer you to call this case as partials. 
Basically it's a case when you desire to put a part of the page to a separate file and this part doesn't represent something independent. Usually it's kind of configuration for local components or components from UI-kit
e.g. have a look on Header in this screen. 

image

[the remark this answers is not in the document]
Comment (divineforest) [reply src/frontend.md:57]: then we could require that such components don't have `.styled` part, OK?

[the remark this answers is not in the document]
Comment (runnez) [reply src/frontend.md:57]: sure

[the remark this answers is not in the document]
Comment (runnez) [reply src/frontend.md:57]: I think If we didn't add an example then it is not allowed, is it?

[the remark this answers is not in the document]
Comment (divineforest) [reply src/frontend.md:57]: better to be explicit here

**Pull request closed without merging**